src/mini.lalrpop

```diff
@@ -88,18 +97,17 @@ BodyDecl: TopLevelDecl = {
 
 TypeDecl: TypeDecl = {
 	"type" <Ident> "=" <Type> => new_type_decl(<>),
+    "type" <Ident> "=" <Type> ";" => new_type_decl(<>),
```

Comment:
The lack of semicolons after type definitions is a historical artifact. It seems more consistent going forward to require a semicolon.  Should we do that?

Reply:
I think we should just go ahead and require a semicolon. I'll commit a change that removes the top rule and fixes ArbOS